eng/verify-nupkgs-exe-version.ps1

```diff
@@ -0,0 +1,8 @@
+Get-ChildItem  S:\p\vstest3\artifacts\packages\Debug\Shipping -Filter vstest.console.exe -Recurse -Force | ForEach-Object {
```

Comment:
Isn't this path linked to your machine setup?

Reply:
Yes I am aware of that, answer to all the same questions below is in the PR description. I just did not want to lose that code.

> Will plug that script later into build, IDK if both those tools are on hosted images, and on everyones computers.